src/client/Microsoft.Identity.Client/Platforms/Android/Broker/AndroidBroker.cs

```diff
@@ -69,10 +98,6 @@ public bool IsBrokerInstalledAndInvokable()
             try
             {
                 await _brokerHelper.InitiateBrokerHandshakeAsync(_parentActivity).ConfigureAwait(false);
-
```

Comment:
Not needed here. This method is for interactive authentication and the username is not required here